day_two/index.js

```diff
@@ -0,0 +1,71 @@
+const testData = require("./test_input");
+
+const problemOne = (noun = 12, verb = 2) => {
+  const fixedOpCode = [testData[0], noun, verb, ...testData.slice(3)];
```

Comment:
smart to not mutate

Reply:
I feel like I try to write pure functions whenever possible, but that I'm working with limited tools. I get the feeling that there are all these other functional programming methods (e.g. currying, which I still don't understand) to do this stuff more idiomatically but I don't know any of it yet. Have you found any good resources on functional JS?